frontend/components/organiser/dashboard/OrganiserChecklist.tsx

```
@@ -63,29 +64,52 @@ export default function OrganiserChecklist() {
+                />
+              ))}
+            </>
+          )}
+          {allItemsChecked && (
+            <>
+              <h1 className="text-center py-6 sm:py-16 font-bold text-xl sm:text-2xl">
+                Good job you have finished setting up ✅ <br></br>
+                Go out there and make more events
+              </h1>
+              <p className="text-[#BABABA] text-end hover:underline hover:cursor-pointer" onClick={resetChecklist}>
+                Reset
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)